comp/api/api/apiimpl/internal/config/endpoint.go

```diff
@@ -50,6 +50,25 @@ func (c *configEndpoint) serveHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (c *configEndpoint) listValuesServeHTTP(w http.ResponseWriter, _ *http.Request) {
+	keys := make([]string, 0, len(c.authorizedConfigPaths))
+	for key := range c.authorizedConfigPaths {
+		keys = append(keys, key)
+	}
+
+	body, err := json.Marshal(keys)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	w.WriteHeader(http.StatusOK)
```

Comment:
nit: `w.Write` writes the `http.StatusOK` header by default iirc but up to you if you want it explicitly listed

Reply:
I know but I'm not a big fan of implicit behavior, I find it clearer to have explicitly what status is used :smile: